stack/keycloak/migration/add-wahlbezirksart-to-user.yml

```
@@ -0,0 +1,47 @@
+id: add wahlbezirksart to user
+author: MrSebastian
+realm: ${SSO_REALM}
+changes:
+  - addUserAttribute:
+      name: wls_all
+      attributeName: wahlbezirksArt
+      attributeValues:
+        - UWB
```

```suggestion
  - addUserAttribute:
      name: wls_wahlbezirksart
      attributeName: wahlbezirksArt
      attributeValues:
        - UWB
        - BWB
```

I don't know enough of these migration scripts, but this seems more correct?

---

a user can only be part of one `wahlbezirksArt`, not Both